src/cmd/run.go

```diff
@@ -0,0 +1,334 @@
+	flags.StringVarP(&runFlags.releaseVersion, "release", "r", "", "Run command inside a toolbox container with the release version")
+	flags.BoolVarP(&runFlags.notty, "notty", "n", false, "Run command inside a toolbox container but do not allocate a tty")
+	flags.BoolVar(&runFlags.emitEscapeSequence, "escape-sequence", false, "Emit an escape sequence for terminals")
+	flags.BoolVar(&runFlags.fallbackToBash, "fallback-to-bash", false, "If requested program does not exist, fallback to bash")
+	flags.BoolVar(&runFlags.promptForCreate, "prompt-for-create", true, "Offer to create a container if it does not exist")
+	flags.BoolVar(&runFlags.pedantic, "pedantic", true, "")
+
+	// These options dont have to be visible to an everyday user
+	flags.MarkHidden("fallback-to-bash")
+	flags.MarkHidden("escape-sequence")
+	flags.MarkHidden("prompt-for-create")
+	flags.MarkHidden("pedantic")
+}
+
+func run(args []string) error {
+	var containerName string = ""
+	var imageName string = ""
+	var commands []string = nil
+
+	// When the release version is specified we want to use it over the container name
+	if runFlags.releaseVersion != "" {
+		if len(args) == 0 {
+			logrus.Fatal("You must provide a command to execute")
+		}
+		commands = args[0:]
+	} else if runFlags.useDefaultContainer {
+		if len(args) == 0 {
+			logrus.Fatal("You must provide a command to execute")
+		}
+		containerName = ""
+		commands = args[0:]
+	} else {
+		if len(args) == 0 {
+			logrus.Fatal("You must provide a name of a container")
+		} else if len(args) == 1 {
+			logrus.Fatal("You must provide a command to execute")
+		}
+		containerName = args[0]
+		commands = args[1:]
+	}
+
+	containerName, imageName, _ = utils.UpdateContainerAndImageNames(containerName, imageName, runFlags.releaseVersion)
+
+	logrus.Debugf("Container: '%s' Image: '%s'", containerName, imageName)
+
+	logrus.Infof("Checking if container '%s' exists", containerName)
+	if !podman.ContainerExists(containerName) {
+		if !runFlags.pedantic {
+			logrus.Errorf("Container '%s' not found", containerName)
+			containers, err := GetContainers()
+			if err != nil {
+				logrus.Fatal("Error while fetching containers")
+			}
+
+			containersCount := len(containers)
+			logrus.Infof("Found %d containers", containersCount)
+
+			if containersCount == 0 {
+				createContainer := false
+
+				if rootFlags.assumeyes {
+					runFlags.promptForCreate = false
+					createContainer = true
+				}
+
+				if runFlags.promptForCreate {
+					var response string
+					for true {
+						fmt.Printf("No toolbox containers found. Create now? [y/N]: ")
+						fmt.Scanf("%s", &response)
+						response = strings.ToLower(response)
+
+						if response == "y" || response == "yes" {
+							createContainer = true
+						}
+
+						break
+					}
+				}
+
+				if !createContainer {
+					logrus.Fatal("A container can be created later with the 'create' command.")
```

Comment:
This is showing up as `FATAL` when `toolbox enter` is invoked without any existing containers. It should be a "normal" message.

Reply:
Nice catch! That should be printed with fmt.Println()